geometry_msgs/Pose start
geometry_msgs/Pose end
---
bool is_found
nav_msgs/Path path
